model_compression_toolkit/target_platform_capabilities/target_platform/targetplatform2framework/target_platform_capabilities.py

```diff
@@ -231,4 +239,28 @@ def is_simd_padding(self) -> bool:
         Returns: Check if the TP model defines that padding due to SIMD constrains occurs.
 
         """
-        return self.tp_model.is_simd_padding
\ No newline at end of file
+        return self.tp_model.is_simd_padding
+
+    def _get_layer_attr_mapping(self, layer: Any, layers_attr_mapping: Dict[Tuple[type], str]) -> str:
```

Comment:
I would rename this method cause one might think that it returns a mapping, and it may confuse.

Reply:
This method is removed once we changed the mapping from a regular dict to a default dict